src/variables.ts

```diff
@@ -57,7 +57,7 @@ export function compileVariableParsing(
   schema: GraphQLSchema,
   varDefNodes: ReadonlyArray<VariableDefinitionNode>
 ): (inputs: { [key: string]: any }) => CoercedVariableValues {
-  const errors = [];
```

Comment:
those arrays without the types are erroring on compilation.

it's also not so easy to give them a type, because they can have literally almost anything inside. they are not strongly typed arrays.